api/src/main/java/org/apache/iceberg/ManifestFile.java

```diff
@@ -41,11 +41,11 @@ public interface ManifestFile {
       "Lowest sequence number in the manifest");
   Types.NestedField SNAPSHOT_ID = optional(503, "added_snapshot_id", Types.LongType.get(),
       "Snapshot ID that added the manifest");
-  Types.NestedField ADDED_FILES_COUNT = optional(504, "added_data_files_count", Types.IntegerType.get(),
+  Types.NestedField ADDED_DATA_FILES_COUNT = optional(504, "added_data_files_count", Types.IntegerType.get(),
```

Comment:
These changes look good to me. I think we can also update the field names. Those won't break anything because the only names that matter are the ones in the read schema.